src/main/java/seedu/duke/parsers/AddParser.java

```diff
@@ -50,10 +51,10 @@
      *                                                      Does not accept " as a valid character.
      *
      * (?<invalid>.*)                                    -- matches [invalid]
-     *                                                      Any other input which do not fit in any of the above
+     *                                                      Any other excess inputs
      */
 
-    private static final String ADD_FORMAT = "(task\\s+\\\"(?<taskName>[^\\\"]+)\\\"(\\s+-m\\s+(?<taskModule>\\w+))?"
+    private static final String ADD_FORMAT = "(task\\s+\\\"(?<taskName>[^\\\"]+)\\\"(\\s+((-m)\\s+|(?<invalidFlag>.*))(?<taskModule>\\w+))?"
```

Comment:
can you check if the regex correctly accepts `add task "task" -d "description"`, based on my online testing, it seems that -d "descriptio" is captured as invalid flag

Reply:
Hmm weird, it seems to work well on my side.

![Screenshot (864)](https://user-images.githubusercontent.com/69583592/161412803-7901dc43-1f11-47ab-be47-9e067ab1c9a2.png)

Reply:
But yes, i do notice if I enter `add task "task" -a "description"`, it is going to eat the last letter.

Not very sure what to add to the regex to fix this.

![Screenshot (865)](https://user-images.githubusercontent.com/69583592/161412896-d0dad641-cd5c-4b42-bc96-c4a350104926.png)

Reply:
maybe consider shifting `(?<taskModule>\w+)` to `(\s+((-m)\s+(?<taskModule>\w+)|(?<invalidFlag>.*)))?`

Reply:
That worked but the output will be `-a CS2101`.

Anyways, Ive changed it to `(\s+((-m|(?<invalidFlag>.*))\s+(?<taskModule>\w+)))?` and now it identify invalid flag to be `-a`. Thanks!

Reply:
![Screenshot (869)](https://user-images.githubusercontent.com/69583592/161414356-895b96f4-53af-4ac1-bc9f-48e6c7d3b0f5.png)